src/routes/staking/staking-form.tsx

```diff
@@ -295,13 +295,14 @@ export const StakingForm = ({
                 </>
               ) : (
                 <>
-                  <p>
+                  <p >
```

Comment:
chore (blocking): pointless space after p
```suggestion
                  <p>
```